### DISCUSSION TITLE
How I found the hottest villages in an Indian state using Planetary Computer & MODIS data

### DISCUSSION THREAD

**Discussion by shijithpk** [microsoft/PlanetaryComputer#193] [Show and tell]
**TLDR**
Used Planetary Computer for an analysis of temperature data
the analysis --> [link](https://shijith.com/blog/kerala-panchayat-temp/)
the code --> [link](https://github.com/shijithpk/hottest-panchayats-kerala)

##############################################

So I'm a data journalist from India, and I live in a village in Kerala, one of India's southern states.

Summer season begins here in a few days, and I wanted to get an idea of how hot villages get here, and how temperatures have evolved in rural Kerala over the past 10 or so years.

There aren't enough weather stations in the state to estimate temperatures for each village. (Over 940 villages here, but only around 100 functional weather stations.)

So I made use of land surface temperature data from MODIS (which is of a relatively high 1 km resolution) and analysed it in Planetary Computer to arrive at figures for each village.

I wrote a [post about it](https://shijith.com/blog/kerala-panchayat-temp/) on my blog. It's a niche topic, but if you have an academic/professional curiosity for rural India, you might find it interesting.

The code for my analysis is available at [this github repo](https://github.com/shijithpk/hottest-panchayats-kerala). It just consists of a jupyter notebook that'll allow you to reproduce the exact results I got.

(Just wish everything could've been done in Planetary Computer itself. The main step of acquiring and analysing MODIS data was actually done using NASA's [appeears api](https://appeears.earthdatacloud.nasa.gov/api/). Tried to do it in Planetary Computer but it kept crashing on me, have [reported the issue](https://github.com/microsoft/PlanetaryComputer/discussions/166) too.)

Pretty sure environment/geo/earth science guys won't be happy with my methodology, especially my use of surface temperature data over 2m air temperature data. But hopefully there's enough of a logic here for the story to be acceptable as a basic analysis from a journalist.

Have any questions, feedback, do let me know, thanks! :)

**Comment by TomAugspurger** [comment]
Thanks for sharing!

I'll try to take a deeper look at https://github.com/microsoft/PlanetaryComputer/discussions/166 when I get a chance.
